test/distributed/test_functional_api.py

```diff
@@ -544,6 +564,17 @@ def test_all_to_all_single_split_sizes_none(self):
         expected = torch.cat(expected)
         self.assertEqual(y, expected)
 
+    @unittest.skipIf(not has_triton(), "Inductor+gpu needs triton and recent GPU arch")
+    @skip_if_lt_x_gpu(WORLD_SIZE)
```

Comment:
Well this may until run for the periodic label enabled.

Reply:
WORLD_SIZE is 2, so I guess it is fine?

Reply:
Oh, NVM, I thought it was 4.